docs/topics/api/signing.rst

```diff
@@ -46,9 +46,11 @@ validation and you will be able to check its status.
             -g -XPUT --form 'upload=@build/my-addon.xpi'
             -H 'Authorization: JWT <jwt-token>'
 
-    :param addon-id the id for the add-on.
-    :param version: the version of the add-on.
-    :form upload: the add-on being uploaded.
+    :param addon-id: The id for the add-on.
```

Comment:
I went looking through the docs to see if there's something else to use and it looks like `:param foo:` would be a URL parameter and `:<json foo:` would be a JSON parameter in the POST data. So I think this is right although I think we haven't used these properly in the past.